core/init.php

```diff
@@ -65,8 +69,8 @@
 
 if ($page != 'install') {
     /*
-     *  Initialise
-     */
+    *  Initialise
```

Comment:
Can we keep the asterisks inline? They look neater imo

Reply:
Good catch, must have been a weird copy/paste issue 😄